bundles/org.openhab.binding.siemensrds/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,189 @@
+		   		<label>Outside Temperature</label>
+		   		<description>Actual outside temperature</description>
+			</channel>
+
+			<channel id="greenLeaf" typeId="greenLeaf">
+		   		<label>Green Leaf Mode</label>
+   				<description>Energy saving level</description>
+			</channel>
+
+			<channel id="statProgramMode" typeId="statProgramMode">
+				<label>T&apos;stat Program (Manual::Auto)</label>
+		   		<description>The program mode of the thermostat (user can override)</description>
+			</channel>
+
+			<channel id="statOccupancyState" typeId="statOccupancyState">
+		   		<label>Occupancy State (Absent::Present)</label>
+   				<description>The occupancy mode of the thermostat (user can override)</description>
+			</channel>
+
+			<channel id="dhwProgramMode" typeId="dhwProgramMode">
+				<label>DHW Program (Manual::Auto)</label>
+		   		<description>The program mode of the domestic water heating (user can override)</description>
+			</channel>
+			
+			<channel id="dhwOffOnState" typeId="dhwOffOnState">
+				<label>DHW Switch State (Off::On)</label>
+		   		<description>The state of the domestic water heating (user can override)</description>
+			</channel>
+
+		</channels>
+
+		<properties>
+			<property name="vendor">Siemens</property>
+			<property name="modelId">RDS</property>
+			<property name="climatixIcPlantId"></property>
+        	</properties>
+        <representation-property>climatixIcPlantId</representation-property>
+
+	</thing-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Measured temperature value</description>
+   		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="setTemperature">
+   		<item-type>Number:Temperature</item-type>
+   		<label>Set Temperature</label>
+		<description>Target temperature setting</description>
+   		<state readOnly="false" pattern="%.1f %unit%" min="5" max="30" step="0.5"/>
+	</channel-type>
+	
+	<channel-type id="heatingOffCooling">
+		<item-type>String</item-type>
+		<label>Heating / Cooling</label>
+   		<description>Heating / Cooling state</description>
+   		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number</item-type>
+		<label>Humidity</label>
+		<description>Measured humidity value</description>
+   		<state readOnly="true" pattern="%.0f %%r.H."/>
+	</channel-type>
+
+	<channel-type id="airQuality">
+		<item-type>String</item-type>
+		<label>Air Quality</label>
+		<description>Room Air Quality</description>
+   		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="greenLeaf">
+		<item-type>String</item-type>
+		<label>Energy savings level </label>
+		<description>Energy savings level</description>
+   		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="statProgramMode">
+		<item-type>Switch</item-type>
+		<label>Thermostat Program Mode (Manual::Auto)</label>
```

Comment:
Please keep label texts as short as possible (1-2 words, if possible)

Reply:
The channel is of type Switch, where the left position of the switch in the Paper UI maps to the thermostat being in the program mode "Manual", and the right position of the switch in the Paper UI maps to it being in the program mode "Auto". So I want to include the "Manual::Auto" part of the text to make that clear to the user. Furthermore the device has two functionalities of the type "Manual::Auto" -- namely "Thermostat" (heating) and Domestic Hot Water, so I think these tests must also be in the label.

=> If you can think of shorter texts that would convey these ideas and avoid confusion, I would be delighted :) but currently I can't think of anything myself :(

PS: this has some background in our earlier conversation on the openHAB.org forum concerning "On/Off/Auto" switches.

Reply:
I don't think we should start adding labels for the two toggle values into the label, which is supposed to represent the whole item. Effectively, a Switch item always has the values ON or OFF, so the question that you have to answer is: "WHAT do you switch on and off?".
So in this case, it would be the "automatic mode" that you can turn on and off, where off immediately means that it is in manual mode.

The reason for the 1-2 word constraint is also very pragmatic: Think of this switch being displayed in BasicUI on a small smartphone screen - it simply won't fit there.

Reply:
Ok.